terraform/variables.tf

```diff
@@ -231,7 +231,7 @@ variable "lock_ttl_clock_skew" {
 }
 
 variable "cloud_scheduler_deadline_duration" {
-  description = "The deadline for job attempts in seconds. If the request handler does not respond by this deadline then the request is cancelled and the attempt is marked as a DEADLINE_EXCEEDED failure. Defaults to 30 minutes (max)"
+  description = "The deadline for job attempts in seconds. If the request handler does not respond by this deadline then the request is cancelled and the attempt is marked as a DEADLINE_EXCEEDED failure. Default is 30 minutes (max)"
```

Comment:
Why this change? I think this is no longer grammatically correct...

Reply:
It previously has a typo `Default is to` so I suggested `Default is`, but I think you're right.

`Defaults to 30 minutes`